src/concepts/backends/alternate-screen.md

```diff
@@ -11,6 +11,51 @@ other terminal content.
 This creates a seamless transition between the application and the regular terminal session, as the
 content displayed before launching the application will reappear after the application exits.
 
+Take this "hello world" program below. If we remove the
```

Comment:
Consider simplifying this code a bit:

- Use `stdout().execute()`
- quit on a delay rather than keys
- hard code the area instead of using layout

Reply:
I made these changes. Removing the quit based on keys allows me to also remove `enable_raw_mode()`.